queries.sql

```diff
@@ -5,4 +5,39 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu'
 SELECT name,escape_attempts FROM animals WHERE weight_kg > 10.5
 SELECT * FROM animals WHERE neutered = true
 SELECT * FROM animals WHERE NOT name = 'Gabumon'
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3
+BEGIN TRANSACTION;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+select * from animals;
+COMMIT TRANSACTION;
+BEGIN TRANSACTION;
+UPDATE ANIMALS
+SET species = 'pokemon'
+WHERE SPECIES IS NULL;
+COMMIT TRANSACTION;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction), Please Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
[Hint] It will look like that:
```sql
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone

```